Headers/AppKit/NSLayoutConstraint.h

```diff
@@ -174,6 +174,8 @@ APPKIT_EXPORT_CLASS
 
 - (void) updateConstraintsForSubtreeIfNeeded;
 
+- (void) _layoutEngineDidChangeAlignmentRect;
```

Comment:
Where will this method be used in the future? Most likely it would be better to move it to a private header file in the Source directory.